plugins/urlscan/komand_urlscan/actions/get_scan_results/action.py

```diff
@@ -47,32 +44,32 @@ def run(self, params={}):
                     "are available when they are requested (typically ~5-10 seconds is sufficient.",
                 )
 
-            if ("message" in js) and ("description" in js):
-                raise PluginException(cause=js["message"], assistance=js["description"])
+            if ("message" in json_response) and ("description" in json_response):
+                raise PluginException(cause=json_response.get("message"), assistance=json_response.get("description"))
 
             raise PluginException(
                 cause="Received an unexpected response from the Urlscan API. ",
                 assistance=f"If the problem persists, please contact support. Response was: {response.text}",
             )
 
         try:
-            ret = js["data"]
-            su = js["task"]["screenshotURL"]
+            data = json_response["data"]
+            screenshot_urls = json_response["task"]["screenshotURL"]
```

Comment:
```suggestion
            data = json_response.get("data")
            screenshot_urls = json_response.get("task", {}).get("screenshotURL")
```

Reply:
@mgrandys-r7 As you can see later goes  catch block for KeyError exception. From my point of view it's allowed to double check format of data returned from the API. I believe that was done to prevent returning empty data to users. So I left this code as it is.